processing/src/test/java/org/apache/druid/java/util/emitter/core/ParametrizedUriEmitterTest.java

```diff
@@ -40,6 +40,7 @@
 import java.util.Map;
 import java.util.Properties;
 
+
```

Comment:
Everything else looks good to me, can you please remove this extra line and squash the 3 commits into one?